pkg/test/runner/runner.go

```diff
@@ -35,13 +31,22 @@ type Runner struct {
 	cmd           string
 	t             *testing.T
 	initialCommit string
+	kptBin        string
+}
+
+func getKptBin() (string, error) {
+	if p := os.Getenv(kptBinEnv); p != "" {
```

Comment:
I realized we can take a commandline arg instead of environment variable (not sure about the benefits though or support both) https://blog.jbowen.dev/2019/08/using-go-flags-in-tests/

Reply:
I think we can use env now and improve it later if we need.